src/props.shadows.css

```diff
@@ -47,4 +47,11 @@
     --shadow-color: 220 40% 2%;
     --shadow-strength: 25%;
   }
+}
+
+[data-theme=dark] {
```

Comment:
this needs to happen here, a place i need to refactor, but here nonetheless: https://github.com/argyleink/open-props/blob/main/build/to-stylesheet.js#L28-L32

add a new line with the `[data-theme="dark"]` selector and put `${dark_propsMeta}` inside.